repo/packages/algebra-plugins/package.py

```diff
@@ -0,0 +1,43 @@
+from spack.package import *
+
+
+class AlgebraPlugins(CMakePackage):
```

Comment:
Similar to VecMem's package, I think this one should inherit from the `CudaPackage` base class as well. (https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/vecmem/package.py#L9) As I would hope that that would take care of setting things like `CMAKE_CUDA_STANDARD` automatically. :thinking:

`CMAKE_SYCL_STANDARD` is a custom variable that we'll need to set ourselves, no question about it. But I wonder if `CMAKE_HIP_STANDARD` will be automatically handled by Spack at one point...  Since standard CMake by now implemented pretty much the same thing that I did in VecMem at the beginning. (It's a long story why we don't yet use their code though.)

Reply:
As I understand it `CudaPackage` just adds support for specifying the CC target as a variant, as well as adding an implicit `cuda` variant to enable or disable CUDA compilation.